networking/advertise-service-ips.md

```diff
@@ -142,6 +144,82 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
    For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
 
+#### Advertise service load balancer IP addresses
+
+The following steps will configure {{site.prodname}} to advertise Service `status.LoadBalancer.Ingress.IP` addresses.
+
+1. Identify the IP ranges to be used for Service LoadBalancer address allocation.
+
+1. Check to see if you have a default BGPConfiguration.
+
+   ```bash
+   calicoctl get bgpconfig default
+   ```
+
+1. Based on above results, update or create a BGPConfiguration.
+
+   **Update default BGPConfiguration**
+   Patch the BGPConfiguration using the following command, adding your own service load balancer IP CIDRs:
+
+   ```bash
+   calicoctl patch BGPConfig default --patch '{"spec": {"serviceLoadBalancer": [{"cidr": "x.x.x.x/16"}]}}'
+   ```
+
+   **Create default BGPConfiguration**
+   Use the following sample command to create a default BGPConfiguration. Add your CIDR blocks for load balancer IPs to be advertised in the `serviceLoadBalancerIPs` field.
+
+   ```bash
+   calicoctl create -f - <<EOF
+   apiVersion: projectcalico.org/v3
+   kind: BGPConfiguration
+   metadata:
+     name: default
+   spec:
+     serviceLoadBalancerIPs:
+     - cidr: x.x.x.x/16
+   EOF
+   ```
+
+   For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
+
+Allocation of Service LoadBalancer addresses is outside the current scope of {{site.prodname}}, but can be implemented with an external controller.
+You may build your own, or use a third-party implementation such as that provided by the MetalLB project.
```

Comment:
You can build your own, or use a third-party implementation like the MetalLB project.